content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -928,7 +928,7 @@ Not supported
 | Microsoft SQL | `varbinary`     |
 | MySQL         | `LONGBLOB`      |
 | MongoDB       | `BinData`       |
-| SQLite        | Not supported   |
+| SQLite        | `BYTES`         |
```

Comment:
Isn't this `BLOB` here then (per the SQL in https://github.com/prisma/prisma/issues/10537#issuecomment-985052256)